### ISSUE TITLE
SWIG 2.0.10 - wrong include path set

### ISSUE THREAD

**Issue by jf---** [ContinuumIO/anaconda-issues#164]
Strange, when installing SWIG 2.0.10 even if `swig -swiglib` is correct the wrond path to SWIG_DIR ( swig includes dir ) is set, see bottom of conda output in bold.

See #48 which is a tangential issue

As such, its not possible to build project with conda SWIG since the include directory is set wrong.
Explicitily overriding in cmake with -DSWIG_DIR=$PREFIX/share/swig/2.0.10 doesnt resolve this, which why its pretty nasty

548 jelleferinga:pythonocc-core$ which swig
/Users/jelleferinga/miniconda/bin/swig
549 jelleferinga:pythonocc-core$ swig -swiglib
/Users/jelleferinga/miniconda/share/swig/2.0.10
550 jelleferinga:pythonocc-core$ 

BUILD START: pythonocc-core-1.0.0-py27_0
Fetching package metadata: ......
Solving package specifications: .
The following NEW packages will be INSTALLED:

    freeimage:                     3.16.0-0     
    freetype:                      2.4.10-1     
    gl2ps:                         1.3.8-0      
    opencascade_community_edition: 0.16.0-py27_4
    openssl:                       1.0.1h-0     
    pcre:                          8.31-0       
    python:                        2.7.8-1      
    readline:                      6.2-2        
    sqlite:                        3.8.4.1-0    
    swig:                          2.0.10-1     
    tbb:                           4.2.5-0      
    tk:                            8.5.15-0     
    zlib:                          1.2.7-1      

Linking packages ...
New string **'/Users/jelleferinga/miniconda/envs/_build/share/swig/2.0.10'** is longer than old string **'/opt/anaconda1anaconda2anaconda3/share/swig/2.0.10'**. Skipping.

**Comment by Juanlu001** [comment]
I got bitten by this today while trying to build a linux64 package with `conda-build`. Any ideas for workarounds or solutions?

**Comment by Juanlu001** [comment]
For further information, I'm using swig 2.0.10.

![Imgur](http://i.imgur.com/RkjPVaK.png)

**Comment by jf---** [comment]
that's a familiar one...
not sure if this is fixed on more recent version of swig ( 3.0.2 is the latest )
but here's a bypass that I use...

mind you that in your case you should probably set it to:

export SWIG_LIB=$PREFIX/share/swig/2.0.10

```bash
export SWIG_LIB=$PREFIX/share/swig/3.0.2
echo "swig <<<"$SWIG_LIB">>>"

# ========== optionally =========

cmake ...
      -DSWIG_DIR=$SWIG_LIB \
```

**Comment by ilanschnell** [comment]
The more recent swig version 3.0.2 has been fixed.  That is, the swig binary gets the actual install prefix fixed when it it is installed using conda, such that it is not necessary to set SWIG_LIB.
